### ISSUE TITLE
[feature] CSV viewing

### ISSUE THREAD

**Issue by citriqa** [qwtel/sqlite-viewer-vscode#30]
quite often i find myself wanting to quickly inspect a CSV file's contents without employing a full-on spreadsheet, and i have not found a good solution for that.
it appears to me that the same interface used for SQLite's tables could easily be used for this purpose; and the complexity of sourcing the data from a CSV file instead isn't very high (i imagine).

do you think this feature has a place in SQLite Viewer?

**Comment by qwtel** [comment]
> do you think this feature has a place in SQLite Viewer?

I don't think it should be part of SQLite Viewer, but for the longest time I've wanted to publish a separat CSV Viewer extension with the same code base, but I havent gotten around to implementing it.

**Comment by ArtemGr** [comment]
Ironically, I would sometimes use a CSV viewer (VSCode “GrapeCity.gc-excelviewer”) with SQLite:

    sqlite3 :memory: -cmd ".mode csv" -cmd ".import analytics.csv analytics" -cmd ".header on" -cmd ".excel" 'select * from analytics'
